### PR TITLE
Refactor and tidy up code formatting

### PR DESCRIPTION
**Pull request by VANQUOC81**
* modified launch.json to use Live Server url
* added comments to toggleBounce function for clarity
* added formatting and comments to explain background looping logic

### CODE REVIEW

**Comment by VANQUOC81** [comment]
@codebreaker131417 please have a look when you have some time.

File: .vscode/launch.json [hunk 1]
```diff
@@ -8,8 +8,8 @@
             "type": "chrome",
             "request": "launch",
             "name": "Launch Chrome against localhost",
-            "url": "http://localhost:8080",
-            "webRoot": "${workspaceFolder}"
+            "url": "http://127.0.0.1:5500/SanroqICT/index.html",
```
Comment (codebreaker131417) [inline .vscode/launch.json:11]: Is the port not dynamically changed every time?

File: SanroqICT/js/stellar.js [hunk 2]
```diff
@@ -584,14 +584,22 @@
...
-			// Reposition backgrounds
-			for (i = this.backgrounds.length - 1; i >= 0; i--) {
-				background = this.backgrounds[i];
-
-				fixedRatioOffset = (background.isFixed ? 0 : 1);
-				bgLeft = (this.options.horizontalScrolling ? (scrollLeft + background.horizontalOffset - this.viewportOffsetLeft - background.startingOffsetLeft + background.parentOffsetLeft - background.startingBackgroundPositionLeft) * (fixedRatioOffset - background.stellarRatio) + 'px' : background.startingValueLeft);
-				bgTop = (this.options.verticalScrolling ? (scrollTop + background.verticalOffset - this.viewportOffsetTop - background.startingOffsetTop + background.parentOffsetTop - background.startingBackgroundPositionTop) * (fixedRatioOffset - background.stellarRatio) + 'px' : background.startingValueTop);
-
+			// This loop repositions backgrounds in a parallax effect
+			// We loop backwards through the backgrounds to make sure that the first layers move slower than the last ones
```
Comment (codebreaker131417) [inline SanroqICT/js/stellar.js:588]: @VANQUOC81 nice.

**Review by codebreaker131417 — COMMENTED** [review]
Review done. See comments.

File: .vscode/launch.json (hunk 1 above)
Comment (VANQUOC81) [reply .vscode/launch.json:11]: Live Server maintains a fixed port of 5500.

File: SanroqICT/js/stellar.js (hunk 2 above)
Comment (VANQUOC81) [reply SanroqICT/js/stellar.js:588]: Np.

**Review by codebreaker131417 — APPROVED** [review]

**Pull request merged**